multi-node/generic/controller-install.sh

```diff
@@ -644,14 +762,26 @@ ExecStartPre=/usr/bin/ln -sf /etc/flannel/options.env /run/flannel/options.env
 EOF
     }
 
-    local TEMPLATE=/etc/systemd/system/docker.service.d/40-flannel.conf
+    local TEMPLATE=/etc/kubernetes/cni/net.d/10-calico.conf
     [ -f $TEMPLATE ] || {
         echo "TEMPLATE: $TEMPLATE"
         mkdir -p $(dirname $TEMPLATE)
         cat << EOF > $TEMPLATE
-[Unit]
```

Comment:
I meant the docker drop in which requires flannel to be running. I believe that should still be needed if not using CNI.